contracts/OfferRegistry.sol

```diff
@@ -165,7 +163,7 @@ contract OfferRegistry is Pausable {
     function toString(address x) internal pure returns (string) {
         bytes memory b = new bytes(20);
         for (uint i = 0; i < 20; i++)
-            b[i] = byte(uint8(uint(x) / (2**(8*(19 - i)))));
+        b[i] = byte(uint8(uint(x) / (2**(8*(19 - i)))));
```

Comment:
Any reason to not explicitly wrap in curly braces?

Reply:
No, and I think solium applied that indent, it's indenting rules are a bit inconsistent. Thanks will address